lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -5063,13 +5063,11 @@ public boolean isItemVisible(SimplePageItem item, SimplePage page, boolean testp
 		if (item.getType() == SimplePageItem.BREAK)
 		    return true;  // breaks are always visible to all users
 		else if (item.getType() == SimplePageItem.PAGE) {
-		  if (!item.isRequired()) {
 		    SimplePage itemPage = getPage(Long.valueOf(item.getSakaiId()));
 		    if (itemPage.isHidden())
 			return false;
 		    if (itemPage.getReleaseDate() != null && itemPage.getReleaseDate().after(new Date()))
-			return false;
-		  }
+			return true;
```

Comment:
Need a opening brace after the if statement, close after. I know lessons does this all over the place, Python style, but still ...